flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+	"only-arches": ["x86_64"]
```

Comment:
so aarch64 can't use this. disappointing.

Reply:
I'm going to add support for this at a later date. I'm gonna make the upstream software support it to, it just hasn't happened yet.

Reply:
you could build from source instead.